src/components/general/ActionsList.tsx

```diff
@@ -26,12 +27,36 @@ const ActionListCategory = styled.div`
   }
 `;
 
+const getValueForSorting = (
+  action: ActionWithEfficiency,
+  sortBy: SortActionsConfig,
+  yearRange: [number, number]
+): number => {
+  if (sortBy.key === 'CUM_IMPACT') {
+    return summarizeYearlyValuesBetween(
```

Comment:
I tested this locally and changed this bit. And then thought I made a mistake. And then brought it back.

Basically we are not using `cumulativeForecastValue` in the UI as it depends on the user selected year range and we are not cycling that through backend.

This logic for the display value is in `ActionCard` and it probably should not be there. We are already mutating the action properties for those "efficiency" actions in `ActionList` so this could go there. 

In the ideal world `ActionCard` would get individual values as props instead of action object.

Reply:
Yeah there's a bunch of logic in `ActionCard`, let's consider refactoring that in the next round of changes